Client/src/App/Studies/StudyActionCreators.js

```diff
@@ -139,7 +139,7 @@ function formatStudies(projectId, questions, recordClasses, answer) {
 
   const unsortedValidRecords = records.valid
     .map(study => Object.assign(study, {
-      disabled: study.projectAvailability && !study.projectAvailability.includes(projectId),
+      disabled: study.access === 'private' && !showUnreleasedData,
```

Comment:
We also seem to allow prerelease studies to be disabled: for example, Jilinde Prospective Cohort is a prerelease study which is disabled on qa.clinepidb but not qa.restricted.clinepidb, because its `project_availability` includes `AllClinEpiDB` and not `ClinEpiDB`.

Are we going to need another flag to handle this case of "disabled prerelease studies"?

Reply:
Thanks for the comment. If I recall correctly, I think we were going to have a specific attribute for this. I will investigate.

Reply:
See https://github.com/VEuPathDB/web-eda/issues/822#issuecomment-1098660382